src/app.js

```diff
@@ -0,0 +1,29 @@
+import './assets/scss/app.scss';
+import $ from 'cash-dom';
+
+
+export class App {
+  initializeApp() {
+    let self = this;
+
+    $('.load-username').on('click', function (e) {
```

Comment:
```suggestion
    $('.load-username').on('click', () => {
```

change function to arrow function and remove unused argument